truss/templates/server/model_wrapper.py

```diff
@@ -252,13 +252,15 @@ async def predict(
         if inspect.isasyncgenfunction(
             self._model.predict
         ) or inspect.isgeneratorfunction(self._model.predict):
-            return self._model.predict(payload)
+            return self._model.predict(payload, is_cancelled_fn)
```

Comment:
We could make the invocation params conditional on whether the truss model's predict method has this parameter (via `inspect`).

Side note on all the other `inspects`s in this part of the code: it might improve performance to make them static attributes instead of re-evaluating them on each invocation.